setup.cfg

```diff
@@ -0,0 +1,28 @@
+[metadata]
+name = flaskr
+version = 1.0.0
+url = https://flask.palletsprojects.com/tutorial/
```

Comment:
Looks like we should change this to the URL for this repo. Also, I think the maintainer and maintainer email should be changed